shin-sforzando/etude-Django
# Virtualize with Docker

## Describe the PR

To close #2 

## Screenshots

<img width="1792" alt="Screen Shot 2021-01-20 at 5 52 17" src="https://user-images.githubusercontent.com/32637762/105091771-b2df8080-5ae3-11eb-9464-3a815c789c06.png">

## Detail of the change

- Add `Dockerfile` and `docker-compose.yml`
- Run `django-admin.py startproject etude .`

## Anticipated impacts

None.

## To reproduce

Steps to check the behavior:

1. Make sure Docker Desktop is running
1. `docker-compose up -d`
1. Check `http://localhost:8000`

## Additional context

Add `CHANGELOG.md` using `cz changelog` .
